framework/src/modules/network/index.js

```diff
@@ -49,12 +49,26 @@ module.exports = class NetworkModule extends BaseModule {
 
 	get actions() {
 		return {
-			request: async action => this.network.actions.request(action),
-			emit: action => this.network.actions.emit(action),
-			getNetworkStatus: () => this.network.actions.getNetworkStatus(),
-			getPeers: action => this.network.actions.getPeers(action),
-			getPeersCountByFilter: action =>
-				this.network.actions.getPeersCountByFilter(action),
+			request: {
+				handler: async action => this.network.actions.request(action),
+				isPublic: false,
+			},
+			emit: {
+				handler: action => this.network.actions.emit(action),
+				isPublic: false,
```

Comment:
do we need to say `isPublic: false` when the default is false?

Reply:
You are right about the action default. My initial thought was that I want people to explicitly think about the publicness of the action they are creating.

Removed the `isPublic:false`.